client/opt/option.go

```diff
@@ -96,17 +103,14 @@ func (o *Option) SetMaxTSOBatchWaitInterval(interval time.Duration) error {
 	}
 	old := o.GetMaxTSOBatchWaitInterval()
 	if interval != old {
-		o.dynamicOptions[MaxTSOBatchWaitInterval].Store(interval)
+		o.dynamicOptions[MaxTSOBatchWaitInterval].CompareAndSwap(old, interval)
 	}
 	return nil
 }
 
 // SetEnableFollowerHandle set the Follower Handle option.
 func (o *Option) SetEnableFollowerHandle(enable bool) {
-	old := o.GetEnableFollowerHandle()
-	if enable != old {
-		o.dynamicOptions[EnableFollowerHandle].Store(enable)
-	}
+	o.dynamicOptions[EnableFollowerHandle].CompareAndSwap(!enable, enable)
```

Comment:
!enable and enable are always different?

Reply:
The goal is to ensure that a swap occurs only when the old value differs from the new value. For instance, if `enable` is `true`, the swap will only take place if the old value of `EnableFollowerHandle` is `false`.